src/Microsoft.TestPlatform.CommunicationUtilities/DataCollectionTestCaseEventSender.cs

```diff
@@ -90,6 +90,12 @@ public void Close()
         public void SendTestCaseStart(TestCaseStartEventArgs e)
         {
             this.communicationManager.SendMessage(MessageType.DataCollectionTestStart, e);
+
+            var message = this.communicationManager.ReceiveMessage();
```

Comment:
>this.communicationManager.ReceiveMessage(); [](start = 26, length = 43)

if this call not sync? why do we need a while loop. for eg in SendTestCaseEnd below i dont see us doing the while loop thing

Reply:
This PR is not fully baked yet. 
I pushed this to verify if Event Log DataCollector Acceptance tests are working or not.